src/sentry/static/sentry/app/views/organizationEventsV2/eventDetails.jsx

```diff
@@ -2,25 +2,56 @@ import React from 'react';
 import styled from 'react-emotion';
 import {browserHistory} from 'react-router';
 import PropTypes from 'prop-types';
+import SentryTypes from 'app/sentryTypes';
 
 import AsyncComponent from 'app/components/asyncComponent';
 import InlineSvg from 'app/components/inlineSvg';
 import withApi from 'app/utils/withApi';
 import space from 'app/styles/space';
 
 import EventModalContent from './eventModalContent';
+import {getQuery} from './utils';
 
 class EventDetails extends AsyncComponent {
   static propTypes = {
-    params: PropTypes.object,
-    eventSlug: PropTypes.string.isRequired,
+    organization: SentryTypes.Organization.isRequired,
+    eventSlug: PropTypes.string,
```

Comment:
We could have a proptype that enforces the eventSlug format

Reply:
Sure!